scripts/install_docker.sh

```diff
@@ -224,6 +224,7 @@ do_install() {
 			$sh_c "apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D"
 			$sh_c "mkdir -p /etc/apt/sources.list.d"
 			$sh_c "echo deb https://apt.dockerproject.org/repo ${lsb_dist}-${dist_version} ${repo} > /etc/apt/sources.list.d/docker.list"
+            $sh_c "sed -i s/xenial/wily/ /etc/apt/sources.list.d/docker.list"
```

Comment:
I really don't like this hack, this would replace ALL releases (trusty included) with wily. We should investigate an if ${dist_version} == "xenial" block down in this code.

I know you didn't have an environment to test this with, so I will take this work item on.